src/Components/BooksList.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function BooksList() {
+  return (
+    <div />
+  );
+}
```

Comment:
- [ ] As part of the project requirements the `BooksList` component should render a list of books using the `<Book>` component.
- [ ] This means you'll have to create a component called `<Book>` that takes the book data as a prop and renders the title and the author it should also have a button for removing books (this doesn't have to be functional for now).